.github/workflows/release.yml

```diff
@@ -82,14 +127,23 @@ jobs:
         id: extract_branch
         run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
       - name: Invoke Oclif workflow
-        if: ${{needs.release.outputs.tag-name != ''}}
+        if: ${{needs.release.outputs.tag-name != '' && github.event.inputs.draft-release != 'true'}}
```

Comment:
Don't we need this running otherwise also? Shouldn't this be an OR condition? The default is set to false